Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -1075,6 +1144,203 @@ public void reallocateMapFragment(final String pureScriptId) {
         });
     }
 
+    @JavascriptInterface
+    public void addRippleCircle(String config) {
```

Comment:
we have ripple classes, why not define these function also to those classes?

Reply:
This can be part of the class handling all these. Any number of circles can be drawn accordingly